Sources/PointFree/Episode/Episode.swift

```diff
@@ -83,75 +82,129 @@ private let videoView = View<Prelude.Unit> { _ in
-  blocks
-    .filter { $0.type == .title && $0.timestamp != nil }
-    .flatMap { block in
-      tocEntryView.view((block.content, block.timestamp ?? 0))
-  }
+  div([`class`([Class.padding([.mobile: [.leftRight: 4, .top: 4]])])],
+    [
+      h6(
+        [`class`([Class.pf.type.title6, Class.pf.colors.fg.gray850, Class.padding([.mobile: [.bottom: 1]])])],
+        ["Chapters"]
+      ),
+      ]
+      <> blocks
+        .filter { $0.type == .title && $0.timestamp != nil }
+        .flatMap { block in
+          tocChapterView.view((block.content, block.timestamp ?? 0))
+    }
+  )
 }
 
-private let tocEntryView = View<(content: String, timestamp: Double)> { content, timestamp in
-  gridRow([`class`([Class.margin.bottom(1)])], [
+private let tocChapterView = View<(content: String, timestamp: Double)> { content, timestamp in
+  gridRow([`class`([Class.margin([.mobile: [.bottom: 1]])])], [
     gridColumn(sizes: [.xs: 10], [
       div([
         a(
-          [href("#"), `class`([Class.pf.colors.fg.white, Class.type.textDecorationNone])],
+          [href("#"), `class`([Class.pf.colors.link.green, Class.type.textDecorationNone])],
           [.text(encode(content))]
         ),
         ])
       ]),
 
     gridColumn(sizes: [.xs: 2], [
       div(
-        [`class`([Class.pf.colors.fg.white, Class.type.align.end, Class.pf.opacity75])],
+        [`class`([Class.pf.colors.fg.purple, Class.type.align.end, Class.pf.opacity75])],
         [.text(encode(timestampLabel(for: timestamp)))]
       )
       ])
     ])
 }
 
-func timestampLabel(for timestamp: Double) -> String {
+private let downloadsView = View<String> { codeSampleDirectory in
+  div([`class`([Class.padding([.mobile: [.leftRight: 4, .top: 3]])])], 
+      [
+        h6(
+          [`class`([Class.pf.type.title6, Class.pf.colors.fg.gray850, Class.padding([.mobile: [.bottom: 1]])])],
+          ["Downloads"]
+        ),
+        img(
+          base64: gitHubSvgBase64(fill: "#FFF080"),
+          mediaType: .image(.svg),
+          alt: "",
+          [`class`([Class.align.middle]), width(20), height(20)]
+        ),
+        a(
+          [
+            href(gitHubUrl(to: GitHubRoute.episodeCodeSample(directory: codeSampleDirectory))),
+            `class`([Class.pf.colors.link.yellow, Class.margin([.mobile: [.left: 1]]), Class.align.middle])
+          ],
+          [.text(encode("\(codeSampleDirectory).playground"))]
+        )
+    ]
+  )
+}
+
+private let creditsView = View<Prelude.Unit> { _ in
+  div([`class`([Class.padding([.mobile: [.leftRight: 4]]), Class.padding([.mobile: [.topBottom: 3]])])],
+      [
+        h6(
+          [`class`([Class.pf.type.title6, Class.pf.colors.fg.gray850, Class.padding([.mobile: [.bottom: 1]])])],
+          ["Credits"]
+        ),
+        p(
+          [`class`([Class.pf.colors.fg.gray850])],
+          [
+            "Hosted by ",
+            a([`class`([Class.pf.colors.link.white]), href("mailto:brandon@pointfree.co")], ["Brandon Williams"]),
```

Comment:
do you think this should just take a string? otherwise you have to use it like:

```swift 
mailto(EmailAddress(unwrap: "brandon@pointfree.co"))
```

not sure what benefits that's giving...